### DISCUSSION TITLE
Mobile view of issues breaks intermittently

### DISCUSSION THREAD

**Discussion by tony19** [community/community#16285] [Mobile]
The mobile view of GitHub Issues on Android Chrome shows an empty box on the right side of the screen, above the comments, which covers the tabs and issue description. The comment boxes themselves are a bit too wide, as their right side touches the very edge of the screen.

https://github.com/vitejs/vite/issues/5447

![Screenshot_20220508-200125_Chrome](https://user-images.githubusercontent.com/26580/167511213-9192cfd4-57a3-4f3c-91d5-26ac454118cb.jpg)

![Screenshot_20220509-181212_Chrome](https://user-images.githubusercontent.com/26580/167513282-e94d2a90-4f4a-4174-9dd3-7ea19da37433.jpg)

**Comment by amrehman** [comment]
Hi there @tony19 ! Sorry to hear that you are having trouble. I will report this bug for you straight away. 

Have you tried giving one of our native mobile apps a go?
https://github.com/mobile

We look forward to your feedback!

**Reply by tony19** [reply]
Thanks for forwarding the issue.

No, I don't use the mobile app, and I don't plan on it.
